website/layout/AutodocsLayout.js

```diff
@@ -654,7 +654,7 @@ var Method = React.createClass({
           </span> || ''}
           {this.props.name}
           <span className="methodType">
-            ({this.props.params && this.props.params.length && this.props.params
+            ({this.props.params && (this.props.params.length !== 0) && this.props.params
```

Comment:
But if `params` can be types other than array, then `this.props.params.length !== 0` is unsafe.